src/TBuffer.cpp

```diff
@@ -4492,7 +4496,6 @@ bool TBuffer::processBig5Sequence(const std::string& bufferData, const bool isFr
                     isToUseReplacementMark = true;
                     break;
                 case 2:
-                    isNonBmpCharacter = true;
```

Comment:
I'd still leave this in place, just in case we **do** get a non-BMP character (say that some new Emojis get added to the existing Big5+), my observation of not getting any character beyond the BMP was just that - and I may be wrong - this will cover things if I am...! :slightly_smiling_face: